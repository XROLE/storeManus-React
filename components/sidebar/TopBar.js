import React, { Fragment } from 'react';

export default function TopBar() {
  return (
    <Fragment>
      <div className="avatar-div-container">
        <div className="avater-div">
          <img src="../../src/img/avatar.jpg" alt="Avatar" className="Avatar" />
          <p className="avatar-label">welcome: Xrole</p>
        </div>
        <div className="filter-div">
          <a href="admin-sales-viewer.html">Filter Sales By: </a>
&nbsp;
          <select name="" id="">
            <option value="">-Select All-</option>
            <option value="">Attendats</option>
            <option value="">Categories</option>
            <option value="">Date</option>
          </select>
&nbsp;
          <select name="" id="">
            <option value="">---- Select All ----</option>
            <option value="Cythia Paul">Cythia Paul</option>
            <option value="Blessing Uche">Blessing Uche</option>
            <option value="Silas Nwokeocha">Silas Nwokeocha</option>
            <option value="Idowu Samson">Idowu Samson</option>
            <option value="Mathew Jade">Mathew Jade</option>
            <option value="Will Smith">Will Smith</option>
            <option value="Cindy Eze">Cindy Eze</option>
            <option value="Simon Peter">Simon Peter</option>
          </select>
                        &nbsp;
          <button className="search-sales-by-attendant-buttton link" type="submit">
            <i className="fas fa-search" />
            <span className="search-text"> &nbsp; Search</span>
          </button>
        </div>
      </div>
    </Fragment>
  );
}
