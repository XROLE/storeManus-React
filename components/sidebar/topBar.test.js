import React from 'react';
import { mount } from 'enzyme';
import TopBar from './TopBar';

describe('TopBar', () => {
  it('Should render just the TopBar component', () => {
    const component = mount(
      <TopBar />,
    );
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
