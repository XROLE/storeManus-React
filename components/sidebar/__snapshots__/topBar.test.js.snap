// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar Should render just the TopBar component 1`] = `
<TopBar>
  <div
    className="avatar-div-container"
  >
    <div
      className="avater-div"
    >
      <img
        alt="Avatar"
        className="Avatar"
        src="../../src/img/avatar.jpg"
      />
      <p
        className="avatar-label"
      >
        welcome: Xrole
      </p>
    </div>
    <div
      className="filter-div hide"
    >
      <a
        href="admin-sales-viewer.html"
      >
        Filter Sales By: 
      </a>
       
      <select
        id=""
        name=""
      >
        <option
          value=""
        >
          -Select All-
        </option>
        <option
          value=""
        >
          Attendats
        </option>
        <option
          value=""
        >
          Categories
        </option>
        <option
          value=""
        >
          Date
        </option>
      </select>
       
      <select
        id=""
        name=""
      >
        <option
          value=""
        >
          ---- Select All ----
        </option>
        <option
          value="Cythia Paul"
        >
          Cythia Paul
        </option>
        <option
          value="Blessing Uche"
        >
          Blessing Uche
        </option>
        <option
          value="Silas Nwokeocha"
        >
          Silas Nwokeocha
        </option>
        <option
          value="Idowu Samson"
        >
          Idowu Samson
        </option>
        <option
          value="Mathew Jade"
        >
          Mathew Jade
        </option>
        <option
          value="Will Smith"
        >
          Will Smith
        </option>
        <option
          value="Cindy Eze"
        >
          Cindy Eze
        </option>
        <option
          value="Simon Peter"
        >
          Simon Peter
        </option>
      </select>
       
      <button
        className="search-sales-by-attendant-buttton link"
        type="submit"
      >
        <i
          className="fas fa-search"
        />
        <span
          className="search-text"
        >
             Search
        </span>
      </button>
    </div>
  </div>
</TopBar>
`;
